packages/mobile/src/icons/VerificationComplete.tsx

```diff
@@ -3,14 +3,14 @@ import Svg, { Path, Circle } from 'svgs'
 
 function VerificationComplete() {
   return (
-    <Svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Add a `testID="VerificationComplete"`

Reply:
How about `testID="VerificationCompleteIcon"` instead? to be clear it's an icon.